CONTRIBUTING.md

```diff
@@ -1,193 +1,169 @@
----
-search:
-  exclude: true
----
```

Comment:
Yes, keeping this content in the search index is important because it helps contributors easily find valuable resources and guidance on contributing to the cp-algorithms project.